### DISCUSSION TITLE
Stop rendering if pre-render script fails

### DISCUSSION THREAD

**Discussion by jkseppan** [quarto-dev/quarto-cli#2474] [Q&A] [answered]
I have a pre-render script of `make`, which takes some input and produces images that I refer to using the `![title](image.png)` format. Sometimes the process fails and I end up with a rendered document with missing images. I would prefer to not get a document at all, so I know something is wrong and can attempt to fix it.

How can I tell Quarto to stop rendering if `make` returns an error code?

**Comment by mcanouil** [comment, marked as answer]
FYI, Quarto is now failing for such cases at least in 1.4.

**Comment by mcanouil** [comment]
FYI, Quarto is now failing for such cases at least in 1.4.
